.github/ISSUE_TEMPLATE/10-bug-performance-issue.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/boronhub/swift/blob/master/KNOWN_ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.
```

Comment:
Please replace all occurrences of `boronhub/swift` to `tensorflow/swift`:

```suggestion
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/swift/blob/master/KNOWN_ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.
```

Same in other files.